cpp/generate_new_journal_alert_stats.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+readonly CONFIG_PATH=/usr/local/var/lib/tuelib/generate_new_journal_alert_stats.conf
+generate_new_journal_alert_stats relbib $(inifile_lookup "$CONFIG_PATH" "" relbib_recipients)
```

Comment:
Instead of sending these via E-Mail, i think it would be better to keep it similar to `export_vufind_redirects`. In that program, the generated statistics are written into a CSV file which can be accessed in the `/mnt/ZE020110/FID-Projekte/Statistik` folder.

Reply:
Done.

Reply:
Now the call of generate_new_journal_alert_stats does not match the usage anymore.
=> relbib_recipient is no longer needed
=> this means we might get rid of the whole config file
=> this also means that we could simply register the ixtheo+relbib conjobs directly in the crontab without thte need of an additional shell script

Reply:
I completely removed this script and updated the crontab instead.

Reply:
The config file is also gone, BTW!